sunpy/net/vso/vso.py

```diff
@@ -271,9 +271,12 @@ def response_block_properties(self):
         """
         s = {a if not a.startswith('_') else None for a in dir(self[0])}
         for resp in self[1:]:
+            if len(s) == 0:
```

Comment:
These fixes look good :+1:

Reply:
should I push another commit that contains changes only in vso file and remove the changes in JSOC.
```python
s.remove(None) # removing none without checking
```
the above line tries to remove None without checking whether set contains None or not which throws KeyError so it is a bug.

Reply:
yeah that would be great :)